client/state/login/actions.js

```diff
@@ -111,7 +121,14 @@ function getErrorFromHTTPError( httpError ) {
 		};
 	}
 
-	const code = get( httpError, 'response.body.data.errors[0]' );
+	code = get( httpError, 'response.body.data.errors.code' );
+	if ( code ) {
+		message = get( httpError, 'response.body.data.errors.message' );
+	} else {
+		// TODO: Remove when we're done with the fallback.
+		code = get( httpError, 'response.body.data.errors[0]', false );
```

Comment:
Why do we set `code` to `false` when it cannot be found now?

I would rather remove that default value to keep the former behavior as otherwise `false` will appear in our stats (we indeed bump each error code encountered), which is less explicit than `undefined`.